packages/cli/src/telemetry/index.ts

```diff
@@ -100,7 +105,9 @@ export class Telemetry {
 			const execTime = new Date();
 			const workflowId = properties.workflow_id;
 
-			this.executionCountsBuffer[workflowId] = this.executionCountsBuffer[workflowId] ?? {};
+			this.executionCountsBuffer[workflowId] = this.executionCountsBuffer[workflowId] ?? {
+				user_id: properties.user_id,
+			};
```

Comment:
Here we are only sending `user_id` if `this.executionCountsBuffer[workflowId]` is nullish. Not familiar with the details of execution counts, but I expect in the normal (non-nullish) case the user ID won't be part of the track payload? Unless it's being already included.

Best way to confirm that it is working is in the PH dashboard.

Reply:
Well my thought process was;
- `executionCountsBuffer` starts empty and is emptied after every pulse
- if the workflowId isn't present, which it won't be after being emptied, then it starts with a `user_id` being present when collecting its first execution count metric for the workflow
- f.ex. line 98 shows where the executionCountsBuffer is cleared again after a pulse is sent

I'll check the dashboard at least and see!